coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/hierarchy/ProposalDevelopmentHierarchyController.java

```diff
@@ -34,6 +34,9 @@ public ModelAndView createHierarchy(@ModelAttribute("KualiForm") ProposalDevelop
             String hierarchyProposalNumber = getProposalHierarchyService().createHierarchy(initialChildProposal, userId);
             displayMessage(ProposalHierarchyKeyConstants.MESSAGE_CREATE_SUCCESS, new String[]{hierarchyProposalNumber});
         }
+
+        form.setEvaluateFlagsAndModes(true);
+        form.setCanEditView(null);
         return getModelAndViewService().getModelAndView(form);
```

Comment:
@jwillia this needs to hide routelog and submission status bar too IMHO ad mentioned in the jira.